Makefile

```diff
@@ -66,51 +61,47 @@ endif
 #----------------
 release: release-android release-ios ##@build build release for Android and iOS
 
-release-android: prod-build-android ##@build build release for Android
-ifdef IN_NIX_SHELL
+release-android: ##@build build release for Android
+	@$(MAKE) prod-build-android && \
```

Comment:
I don't get the point of this `@$(MAKE)`. Why are you adding this?

Reply:
oh, this should have been reverted to the old way. Thanks for catching it.

Reply:
I recall why it's done this way. If it is done the old way, it is going to instantiate the nix-shell twice, taking much more time. So by doing this we're optimizing so that a single nix-shell is opened. Make sense?

Reply:
No, the reverse is happening. You are running `make ${target}`, then the target runs another `make` which creates two nested nix shells.

Reply:
the second make will know it is already inside a nix-shell though, right?

Reply:
Ah yes, that's true, but also redundant.